src/main/java/org/broadinstitute/hellbender/tools/exome/CreateAllelicPanelOfNormals.java

```diff
@@ -70,6 +75,11 @@
     protected Object doWork() {
         validateArguments();
 
+        if (!new HDF5Library().load(null)) {  //Note: passing null means using the default temp dir.
+            throw new UserException.HardwareFeatureException("Cannot load the required HDF5 library. " +
+                    "HDF5 is currently supported on x86-64 architecture and Linux or OSX systems.");
+        }
+
```

Comment:
This code snippet is repeated in `AllelicCNV` and `CalculatePulldownPhasePosteriors` classed. Could we move it to some Utils class?